pkg/server/server.go

```diff
@@ -115,19 +115,30 @@ func (s *Server) Run(ctx context.Context) error {
 
 		klog.Infof("Finished start kube informers")
 
-		if err := systemcrds.Bootstrap(
-			goContext(ctx),
-			s.ApiExtensionsClusterClient.Cluster(SystemCRDLogicalCluster),
-			s.ApiExtensionsClusterClient.Cluster(SystemCRDLogicalCluster).Discovery(),
-			s.DynamicClusterClient.Cluster(SystemCRDLogicalCluster),
-		); err != nil {
+		if err := wait.PollInfiniteWithContext(goContext(ctx), time.Second, func(ctx context.Context) (bool, error) {
+			if err := systemcrds.Bootstrap(ctx,
+				s.ApiExtensionsClusterClient.Cluster(SystemCRDLogicalCluster),
+				s.ApiExtensionsClusterClient.Cluster(SystemCRDLogicalCluster).Discovery(),
+				s.DynamicClusterClient.Cluster(SystemCRDLogicalCluster),
+			); err != nil {
+				klog.Errorf("failed to bootstrap system CRDs: %v", err)
```

Comment:
when and how can this actually fail?

Reply:
network error, wrong manifests, any kind of 50x

Reply:
> a network error

aren't we using the local client?

> wrong manifest 

this can never succeed